package.json

```diff
@@ -75,14 +93,15 @@
     "test": "nyc mocha build/test",
     "lint": "gts check && eslint samples/*.js samples/**/*.js",
     "clean": "gts clean",
-    "compile": "tsc -p . && cp src/*.json build/src && cp src/*.js build/src",
+    "compile": "pbjs -t json ./protos/google/longrunning/operations.proto -p ./protos > pbjs-genfiles/operations.json && tsc -p . && cp src/*.json build/src && cp src/*.js build/src",
     "fix": "gts fix && eslint --fix samples/*.js samples/**/*.js",
-    "prepare": "npm run compile && node ./build/tools/prepublish.js",
+    "prepare": "tsc -p tools && node ./build/tools/prepublish.js && npm run compile",
```

Comment:
is there a good reason to compile `tools` separately from other typescript files?

Reply:
The reason we did this is with our change to generating JSON files out of our protofiles as part of our compile step, we need to have the proto files available and want to run prepublish.js first.

Reply:
So... I thought a little bit more about it. @JustinBeckwith suggested we just go ahead and check in the JSONs for the client libraries, simplifying the prepare step. So @LibanOdowa how about if we just go ahead and checkin this `operations.json` into the `protos` folder, getting rid of `pbjs-genfiles` at all? We can still have an NPM script

```
    "compile-protos": "pbjs -t json ./protos/google/longrunning/operations.proto -p ./protos > protos/operations.json"
```

just in case if anyone needs to regenerate it but the JSON will be checked in and this script won't be called automatically. What do you think?

Reply:
I also think checking in the generated JSON is fine, it's going to need to be published to npm with the module any ways right?

Reply:
Sounds good, done.